### ISSUE TITLE
Is there anyone that succesfully run the code ?

### ISSUE THREAD

**Issue by MyraBaba** [SamurAIGPT/privateGPT#62]
Hi,

I downloaded reqirements and npn dependencies but many errors..

UI : localhost:3000 opening but model downloading gives error. also document upload also gives error.

**Comment by kingsky-z** [comment]
Yes , I also met the same problem.  but after i  download the models manually  , it runs.

**Comment by MyraBaba** [comment]
I downloaded the model and put the models folder. 
gptj_model_load: loading model from 'models/ggml-gpt4all-j-v1.3-groovy.bin' - please wait ...

still errors as shown in the video

https://github.com/SamurAIGPT/privateGPT/assets/17505439/0b63d483-037f-4828-806a-982ba967c2dd

**Comment by MyraBaba** [comment]
@kingsky-z  yes it runs . It was the silly problem.. I tried to load doc from different computer but server looking the local path.

additionally other than english lang is not supported I assume

**Comment by vdayrell** [comment]
here is working well, just follow step by step... but I don't know why the answer is not so precise like martinez' one. Still trying to figure what is the difference.

**Comment by djibe** [comment]
Working well in a few minutes.

**Comment by navicstein** [comment]
it takes time to give a reply
